astro/src/content/docs/release-notes/index.mdx

```diff
@@ -13,17 +13,119 @@ import ReleaseNoteHeading from 'src/components/docs/release-notes/ReleaseNoteHea
+
+   1. If you have a db migration, add the <DatabaseMigrationWarning /> to the release note.
+   2. If you have a db migration, and add the version to operate/deploy/upgrade.mdx in the `Out-of-Band Database Upgrades` section.
+   3. Add the following sections, removing any that are empty, but keeping this order:
+     - Known Issues
+     - Security
+     - Changed
+     - New
+     - Fixed
+     - Enhancements
+     - Internal
+*/}
 
 Looking for release notes older than 1.23.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
+<ReleaseNoteHeading version='1.52.0' releaseDate='August 8th, 2024' />
+
+<GeneralMigrationWarning>
+  When using the User Registrations API, the `data` field for the FusionAuth application with Id `3c219e58-ed0e-4b18-ad48-f4f92793ae32` may now contain a `preferences` object. This object is reserved and should not be modified.
+</GeneralMigrationWarning>
+
+<DeprecationWarning>
+  **Group Member API**
+
+  The `user` field on the Group Member API responses is being deprecated.
+
+  This field was not documented, and has never been populated on the API response. However, because this field was generated and part of the domain in FusionAuth client libraries, we are providing a deprecation notice in case this may affect your integration. Client library users should remove references at your earliest convenience.
+
+  Removal of this field is targeted for the end of 2024.
+
+  **Docker architectures**
+
+  We are planning to discontinue publishing Docker images for the following architectures: `linux/arm/v7`, `linux/ppc64le`, and `linux/s390x`. The rationale behind this decision is that we do not believe they are actively being used, and we would like to move to the GraalVM Java distribution which does not provide builds for these architectures.
+   * https://hub.docker.com/r/fusionauth/fusionauth-app
+
+  We plan to stop publishing docker images for these architectures at the end of 2024. If you are actively using any of these architectures, please let us know how this could affect you by contacting support or reaching out to sales.
+</DeprecationWarning>
+
+### Security
+
+* When detecting impossible travel or similarly suspicious login events, it is possible that not all device trust cookies were correctly revoked.  These are now automatically revoked.
+  * Resolves [GitHub Issue #2753](https://github.com/FusionAuth/fusionauth-issues/issues/2753)
+
+### New
+* A free community license is now available, which adds WebAuthn (Passkeys) to the community edition. All those with a Community license will now find a license key in their [FusionAuth account plan](https://account.fusionauth.io/account/plan) page. And there was much rejoicing! 🥳
```

Comment:
I don't mind them, but this release doubles the number of emoji we have in the release notes. Just wanted to call out the change in tone.

Reply:
Noted.